orchestrator/careplanservice/service_test.go

```diff
@@ -46,6 +53,100 @@ func TestService_Proxy(t *testing.T) {
 	require.Equal(t, fhirServerURL.Host, capturedHost)
 }
 
+type OperationOutcomeWithResourceType struct {
+	fhir.OperationOutcome
+	ResourceType *string `bson:"resourceType" json:"resourceType"`
```

Comment:
why the `bson` tag?

Reply:
That's how all fhir.* structs do this, so I gathered it we also supported that.